File: src/test/java/kitchenpos/fixture/OrderTableFixtures.java
package kitchenpos.fixture;

import kitchenpos.domain.OrderTable;

import java.util.UUID;

public class OrderTableFixtures {

    public static OrderTable createOrderTable(String name, boolean occupied, int numberOfGuests) {
        OrderTable orderTable = new OrderTable();
        orderTable.setId(UUID.randomUUID());
        orderTable.setName(name);
        orderTable.setOccupied(occupied);
        orderTable.setNumberOfGuests(numberOfGuests);
        return orderTable;
    }
}

File: src/test/java/kitchenpos/testsupport/OrderTableFixtures.kt
package kitchenpos.testsupport

import java.util.UUID
import kitchenpos.domain.OrderTable

object OrderTableFixtures {
    fun createOrderTable(): OrderTable {
        return OrderTable().apply {
            this.id = UUID.randomUUID()
            this.name = "test-order-table-name"
            this.numberOfGuests = 0
            this.isOccupied = false
        }
    }
}